lib/zlib.js

```diff
@@ -30,12 +30,12 @@ const {
 } = require('internal/errors').codes;
 const Transform = require('_stream_transform');
 const {
-  deprecate,
-  types: {
+  deprecate
+} = require('internal/util');
+const {
     isAnyArrayBuffer,
     isArrayBufferView
```

Comment:
Almost done 👍.

It needs one more step to be landed: ensure the indent should be 2 here.

![Screen Shot 2019-04-01 at 11 22 54 PM](https://user-images.githubusercontent.com/23313266/55339521-6cb3ca00-54d5-11e9-8325-c0e8f77df292.png)